Jenkinsfile

```diff
@@ -167,7 +167,7 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
                 try {
                     sh "cp ${env.PRIVATE_KEY} ${WORKSPACE}/key-build"
                     sh "cp ${env.PUBLIC_KEY} ${WORKSPACE}/key-build.pub"
-                    sh "make -j4 V=s ${params.ALL_PACKAGES ? 'IGNORE_ERRORS=m' : ''}"
+                    sh "make -j1 V=s ${params.ALL_PACKAGES ? 'IGNORE_ERRORS=m' : ''}"
```

Comment:
What is the need of reducing the thread count?  If -j4 fails, the try-catch would allow building again with -j1 anyway